pkg/infrastructure/database/postgres_metrics_repository_test.go

```diff
@@ -66,11 +66,66 @@ func TestPostgresMetricsRepository_Count_Unit(t *testing.T) {
 	}
 
 	// When there is a row
-	n, err = pgr.Count("UUID")
+	n, err = pgr.CountAll("UUID")
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	// Then
 	assert.Equal(t, 2, n, "Bad count")
 }
+
+func TestPostgresMetricsRepository_FindRequestsByDay_Unit(t *testing.T) {
```

Comment:
Je sais que ca va pas être hyper lisible mais je suis pas super fan de la combinaison Camel Case + Snake Case. Aussi l'intention n'est pas très clair dans le nom du test. Pourquoi ne pas tenter un `TestRetrieveRequestsByDayInPostgresMetricRepository` ?

Reply:
It's Go's convention to have : `TestClassName_Method`, I added `_Unit` to handle running unit and integration tests